hack/fuse-demo/verify_coord_bundle.sh

```diff
@@ -11,12 +11,16 @@ start_timestamp=$(cat /tmp/datamon_fuse_demo_coord_start_timestamp)
 
 verify_datamon_timestamp() {
     timestamp="$1"
-    timestamp_to_parse=$(echo "$timestamp" | sed 's/\.[^ ]*//g' | sed 's/ *[^ ]*$//')
+    timestamp_to_parse=$(echo "$timestamp" | \
+                             sed 's/\.[^ ]*//g' | sed 's/ *[^ ]*$//')
     epoch_timestamp=$(date -jf '%Y-%m-%d %H:%M:%S %z' "$timestamp_to_parse" '+%s')
-    sec_from_start=$(echo "$epoch_timestamp - $start_timestamp" |bc)
+    start_timestamp_local=$(date -jf  '%s' "$start_timestamp" \
```

Comment:
mmh I don't know the `-j` flag for date. This is most likely non standard.

Reply:
When it comes to parsing dates into timestamps, I usually do something like `date --date "${timestamp}" '+%s'`(GNU date). Wouldn't that help clarifying this section?

Reply:
unclear why gnu is more or less of a standard than freebsd